test.js

```diff
@@ -1,57 +1,56 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+let stream;
+let output;
```

Comment:
But you're right, I'm looking into AVA for the first time.
I'll fix the context problem.